scripts/UPDATE.sh

```diff
@@ -120,11 +120,14 @@ SuperBuild(){
   cd $SIRF_SRC_PATH
   if [ ! -d SIRF-SuperBuild ] 
   then
-    git clone https://github.com/CCPPETMR/SIRF-SuperBuild.git
+    git clone https://github.com/SynerBi/SIRF-SuperBuild.git
     cd SIRF-SuperBuild
   else
     cd SIRF-SuperBuild
+    # update the remote repository after rename to SynerBI
+    git remote set-url origin https://github.com/SynerBI/SIRF-SuperBuild.git
     git fetch
+    git pull
```

Comment:
can't do an explicit `pull`. The rest of the script will take care of it. `fetch` updates the content of the remote branches. `pull` would do `fetch` and `merge`

Reply:
Sure? I added it just because fetch wouldn't suffice. I'll try again.

Reply:
It seems to me that we only request:
1. fetch
1. checkout to a different tag

I don't see how the rest of the script does a pull of the `SyneRBI_VM` if we don't change branch. Am I wrong?

Reply:
I have thought more about this. I've created #153. Let's not fix this in this PR though.

Note that a  `pull` *before* the checkout doesn't change anything in the first 2 cases mentioned in #153, and potentially updates the wrong local branch.

Reply:
I removed the `pull`